src/components/Menu.tsx

```diff
@@ -11,17 +11,9 @@ export const MenuItem:React.FunctionComponent<{label: string, href: string}> = (
 };
 
 export const Menu:React.FunctionComponent<{children: React.ReactNode}> = (props) => {
-  const [open, setOpen] = React.useState(false);
-  React.useEffect(() => {
-    const listener = () => open && setOpen(false);
-    document.addEventListener('click', listener);
-    return () => document.removeEventListener('click', listener);
-  });
   const classes = [styles.Menu];
-  if (open) classes.push(styles.open);
   return (
     <div className={classes.join(' ')}>
-      <div className={styles.icon} onClick={() => setOpen(!open)}>☰</div>
```

Comment:
Can we keep the hamburger menu on mobile (i.e. viewport width  600px)? This is causing viewport sizing issues on mobile.

Maybe have a hamburger menu component and a desktop meny component.